components/core/src/package/install.rs

```diff
@@ -22,9 +22,9 @@ use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
 pub use types::package_install::*;
+use super::{Identifiable, MetaFile, PackageIdent, Target, PackageTarget};
```

Comment:
Using super here feels less clear than the former path, which used `package`. Why the change?